extensions/cornerstone-dicom-seg/src/getToolbarModule.ts

```diff
@@ -33,6 +65,24 @@ export function getToolbarModule({ servicesManager }: withAppTypes) {
           };
         }
 
+        const activeRepresentations = segmentationService.getSegmentationRepresentations(
+          viewportId,
+          {
+            segmentationId: activeSegmentation.segmentationId,
+            type: segmentationRepresentationType,
+          }
+        );
+        if (!activeRepresentations?.length) {
```

Comment:
i prefer separate eval for segmentation type (the one you have in the top)

Reply:
Sure.